src/utils/environment.ts

```diff
@@ -7,22 +7,32 @@
 import Constants from 'expo-constants';
 
 const ENV = {
+  local: {
+    apiUrl: 'http://097ab81b.ngrok.io',
```

Comment:
Replace with env-dev before check-in.

Reply:
Looks like this still hasn't been changed.

Reply:
oops, I updated it, then changed it back to test some of my other changes. updating again!